custom_components/tahoma/sensor.py

```diff
@@ -75,8 +78,14 @@ def unit_of_measurement(self):
         states = self.tahoma_device.active_states
 
         if CORE_TEMPERATURE_STATE in states:
-            # TODO Retrieve core:MeasuredValueType to understand if it is Celsius or Kelvin
-            return TEMP_CELSIUS
+            return {
+                "core:TemperatureInCelcius": TEMP_CELSIUS,
```

Comment:
Should we add a note that they have a typo here? (and that it is not in our code). Or add the correctly spelled one as well, to be sure.

Reply:
added the correct one just in case